### DISCUSSION TITLE
Bitcoin - wrong send value in confirm screen

### DISCUSSION THREAD

**Discussion by gomesalexandre** [phantom/docs#293] [Help & Support] [answered]
### Summary

Currently seeing wrong send value in confirm screen for BTC sends - it uses the fees as a send value 

For reference, this is our pst `.data.globalMap.unsignedTx' path (can't paste the full psbt here for some reason as it doesn't serialize well):

<img width="454" alt="image" src="https://github.com/user-attachments/assets/0b56100c-a55c-44e2-808e-cf567d0ce8a5">

### Steps to Reproduce

- Perform a SegWit native (the only currently supported script type) send
- Notice send value is off
- Notice expending it correctly shows the right outputs, including the actual send

<img width="698" alt="image" src="https://github.com/user-attachments/assets/a2617a1a-0d77-4fac-8203-43d46b03f1d9">

![image](https://github.com/user-attachments/assets/2a6dc3b0-4a7b-4c6d-8f03-c60fd02576df)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @gomesalexandre,

These balances seem correct since you are sending to the same wallet you are sending from.

**Reply by gomesalexandre** [reply]
Oh I see so that's not so much of taking your input value and using that as a value but rather doing a relative balance - balance after outputs - gas! It makes sense then, given there is no xpub support, I guess self-sends would be quite exotic of a flow and probably used by devs only.

Thanks for the disambiguation @adamdelphantom! That's actually quite a nice way to represent it 🙏🏽
